example.nxc

```diff
@@ -6,18 +6,28 @@
 task main()
 {
     SetSensorNXTViSy(PORT);
-    NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    long err = NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    if (err != NO_ERR) {
+        NumOut(0, LCD_LINE4, err);
+        while(1);
```

Comment:
@vargac again, I'd very much prefer if you could put the semicolon (`;`) on its own line. If for no other reason, then just to make @burlito happy.